src/translations/en/renderer.json

```diff
@@ -156,8 +156,9 @@
     "syncStatus": {
       "syncConnecting": "Connecting",
       "fullySynced": "Fully Synced",
-      "syncingBlocks": "Syncing Blocks {{percentage}}%",
-      "syncingState": "Syncing State {{percentage}}%",
+      "syncingBlocks": "Syncing Blocks: {{percentage}}%",
+      "syncingState": "Syncing State: {{processed}} of {{total}}",
+      "syncingStateDescription": "Mantis Wallet is fetching all accounts state, including balances and smart contract codes",
```

Comment:
What does "code" mean here? Status code?

Reply:
It refers to the code which is executed by the smart contract.

Reply:
Not sure what would be a better phrasing in this case, let's just roll with it.